### ISSUE TITLE
Incorrect `aria-expanded` on language button after language selection

### ISSUE THREAD

**Issue by sculpt0r** [ckeditor/ckeditor4#5144]
## Type of report

Bug

## Provide detailed reproduction steps (if any)

1. Open [CKE4 with language plugin](https://ckeditor.com/docs/ckeditor4/latest/examples/language.html)
2. Inspect the language toolbar button and find its `aria-expanded` attribute. At this moment it is set to `false`
3. Use the language button and select some language.

### Expected result

*What is the expected result of the above steps?*

The `aria-expanded` attribute should be set to `false` since the submenu is folded. Also, screen readers shouldn't read this button at the current state as `expanded`.

### Actual result

*What is the actual result of the above steps?*

The `aria-expanded` attribute is set to `true`. Screen readers read this button as expanded when any language is selected.

## Other details

* Browser: -
* OS: -
* CKEditor version: `4.18.0` (reproducible even back to `4.13.0` - so no regression)
* Installed CKEditor plugins: language

**Comment by Comandeer** [comment]
The issue is not as simple as it looks and it's present on every menu button in the toolbar (e.g. "Text color"). It originates from the fact that menu buttons are in the "on" state in two situations:

1.  the menu is open,
2.  the content represents one of the menu options (e.g. the gray text would make the "Text Color" button switch into the "on" state).

At the moment we treat the "on" state as the indication of an open menu → [https://github.com/ckeditor/ckeditor4/blob/dabfeff5e47d92092d195ff6ba865662787c99c0/plugins/button/plugin.js#L347-L350](https://github.com/ckeditor/ckeditor4/blob/dabfeff5e47d92092d195ff6ba865662787c99c0/plugins/button/plugin.js#L347-L350)

We would need to come up with a better way to detect it.
